src/define.js

```diff
@@ -72,8 +72,8 @@ function compile(Hybrid, descriptors) {
 
     if (config.connect) {
       callbacks.push(host =>
-        config.connect(host, key, () => {
-          cache.invalidate(host, key);
+        config.connect(host, key, options => {
+          cache.invalidate(host, key, options);
```

Comment:
```suggestion
          cache.invalidate(host, key,{ force: options && options.force === true });
```

As I saw the change in the `children.js` file if we don't do type checking it will mean that this PR has a breaking change, which I would like to avoid. With the above code, even if the user passed something into the `invalidate()` callback, it would have to be an object with `force` property equals true to make a change, so it marginally possible.